Src/LiquidProjections/Projector.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace LiquidProjections
+{
+    public sealed class Projector : Projector<ProjectionContext>
+    {
+        public Projector(IEventMapBuilder<ProjectionContext> eventMapBuilder) 
+            : base(eventMapBuilder, (transaction, @event) => new ProjectionContext
+            {
+                StreamId = transaction.StreamId,
+                TimeStampUtc = transaction.TimeStampUtc,
+                Checkpoint = transaction.Checkpoint,
+                EventHeaders = @event.Headers,
+                TransactionHeaders = transaction.Headers
+            })
+        {
+        }
+    }
+
+    public class Projector<TContext>

```

Comment:
So is this the basic projector that you can next to more specialized versions for RavenDB and NHibernate?